src/ucp/core/ucp_worker.c

```diff
@@ -642,7 +642,9 @@ ucs_status_t ucp_worker_arm(ucp_worker_h worker)
         wakeup = worker->wakeup.iface_wakeups[tl_id];
         if (wakeup != NULL) {
             status = uct_wakeup_efd_arm(wakeup);
-            if (status != UCS_OK) {
+            if (UCS_ERR_BUSY == status) {
+                result_status = status;
```

Comment:
if we return OK and one of the transports had ERR_BUSY, the user would go to epoll_wait() and potentially miss events from that transport